aspnetcore/data/ef-mvc/crud.md

```diff
@@ -153,7 +153,7 @@ The code in *Views/Students/Create.cshtml* uses `label`, `input`, and `span` (fo
 
 Run the page by selecting the **Students** tab and clicking **Create New**.
 
-Enter names and an invalid date and click **Create** to see the error message.
+Enter names and a date. Try entering an invalid date if your browser lets you do that. (Some browsers force you to use a date picker.) Then click **Create** to see the error message.
```

Comment:
Consider either having them use another field or mentioning 
Chrome and Edge use the date picker. Currently, Firefox and IE do not.